lib/embed/viewer/image_x.rb

```diff
@@ -1,3 +1,5 @@
+require 'embed/download_panel'
```

Comment:
:+1: Good :eyes:

I think this was just leftover from the `DownloadPanel` refactor.  I'll check that it all works w/o it and remove it.